packages/mobile/src/components/audio-rewards/IconAudioBadge.tsx

```diff
@@ -0,0 +1,29 @@
+import { ComponentType } from 'react'
+
+import { SvgProps } from 'react-native-svg'
+
+import IconBronzeBadge from 'app/assets/images/IconBronzeBadge.svg'
+import IconGoldBadge from 'app/assets/images/IconGoldBadge.svg'
+import IconPlatinumBadge from 'app/assets/images/IconPlatinumBadge.svg'
+import IconSilverBadge from 'app/assets/images/IconSilverBadge.svg'
+
+import { AudioTier } from './audioTier'
+
+const audioTierMap: Record<AudioTier, ComponentType<SvgProps> | null> = {
```

Comment:
nit - should we UPPER_SNAKE_CASE constants like these? `AUDIO_TIER_MAP`

Reply:
yeah def agree!